README.md

```diff
@@ -1,25 +1,3 @@
 # The BFpack Module
```

Comment:
Is there a reason we are deleting the readme? I actually like it quite a bit---it would be nicer if the other modules also had better readmes maybe with some future development plans etc...

Reply:
https://github.com/jasp-stats/jaspMetaAnalysis
https://github.com/jasp-stats/jaspMixedModels
https://github.com/jasp-stats/jaspSurvival
Should I keep going...